internal/datastreams/hash_cache.go

```diff
@@ -19,17 +19,23 @@ type hashCache struct {
 	m  map[string]uint64
 }
 
-func getHashKey(edgeTags []string, parentHash uint64) string {
+func getHashKey(edgeTags, processTags []string, parentHash uint64) string {
 	var s strings.Builder
 	l := 0
 	for _, t := range edgeTags {
 		l += len(t)
 	}
+	for _, t := range processTags {
+		l += len(t)
+	}
 	l += 8
 	s.Grow(l)
 	for _, t := range edgeTags {
 		s.WriteString(t)
 	}
+	for _, t := range processTags {
```

Comment:
should that be sorted in order to have the same ordering? It will likely change the hash

Reply:
the slice is already sorted when its built: https://github.com/DataDog/dd-trace-go/blob/main/internal/processtags/processtags.go#L84-L89